proxy/http/proxy/server.py

```diff
@@ -32,6 +32,7 @@
 from ..plugin import HttpProtocolHandlerPlugin
 from ..exception import HttpProtocolException, ProxyConnectionFailed
 from ..parser import HttpParser, httpParserStates, httpParserTypes
+# from ..parser.tls import TlsParser, TlsHandshake, tlsContentType, tlsHandshakeType
```

Comment:
Can you send a separate PR for `TlsParser` utility also.  We'll still need it later when we want to support HTTPS connection pooling across all connection interfaces.

Reply:
Okay